comfy_cli/command/transfer.py

```
@@ -0,0 +1,560 @@
+                        hint="check the prompt_id and ensure the job has completed",
+                        details={"prompt_id": prompt_id},
+                    )
+                    raise typer.Exit(code=1)
+            except Unauthenticated:
+                renderer.error(
+                    code="download_job_not_found",
+                    message=f"Job {prompt_id} not found in local state files and cloud auth is missing",
+                    hint="run 'comfy cloud login' or check the prompt_id",
+                    details={"prompt_id": prompt_id},
+                )
+                raise typer.Exit(code=1)
```

Pylint (4.0.5)

[warning] Consider explicitly re-raising using 'except Unauthenticated as exc' and 'raise typer.Exit(code=1) from exc'

(W0707)